src/helpers/files.js

```diff
@@ -0,0 +1,56 @@
+import Vue from 'vue'
+import FileListInfo from '../views/FileListInfo.vue'
+import axios from '@nextcloud/axios'
+import { generateOcsUrl } from '@nextcloud/router'
+
+const FilesCollectivesPlugin = {
+	el: null,
+
+	async getCollectivesFolder() {
+		const response = await axios.get(generateOcsUrl('apps/collectives/api/v1.0/settings/user/user_folder'))
+		return response.data.ocs.data
+	},
+
+	attach(fileList) {
+		if (fileList.id !== 'files' && fileList.id !== 'files.public') {
+			return
+		}
+
+		this.el = document.createElement('div')
+		this.el.id = 'files-collectives-info-wrapper'
+		fileList.registerHeader({
+			id: 'collectives',
+			el: this.el,
+			render: this.render.bind(this),
+			// Higher order than Text rich workspaces
+			order: -1,
+		})
+	},
+
+	async render(fileList) {
```

Comment:
I'm a bit unsure whether it's acceptable to make `render()` a async function. While it works as expected, I have a bad gut feeling about it. What do you think @juliushaertl @max-nextcloud?

Maybe you have another idea how to implement the async `getCollectivesFolder()` function and still make sure that `render()` doesn't pass a promise in `propsData`?

Reply:
You could also pass the collective folder as initial state in the Listener for the files additional scripts. Then you could directly perform the check without waiting for another request.

Reply:
Oh, great idea, thanks for the hint! To be honest I didn't know about initial state before :see_no_evil:

Implemented now in https://github.com/nextcloud/collectives/pull/461/commits/6fd1f7304756e596be2272ee15f0ceb4637a0c4b